npm/ls-lint/bin/index.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+var spawn = require('child_process').spawn;
+var path = require('path');
+
+var command_args = process.argv.slice(2);
+
+function spawnCommand(binaryExecutable) {
+  var child = spawn(
+    path.join(__dirname, binaryExecutable),
+    command_args,
+    { stdio: [process.stdin, process.stdout, process.stderr] }
+  );
+
+  child.on('close', function(code) {
+    if (code !== 0) {
+      process.exit(1);
+    }
+  });
+}
+
+if (process.platform === 'darwin') {
+  spawnCommand('ls-lint-darwin');
+} else if (process.platform === 'linux') {
+  spawnCommand('ls-lint-linux');
+} else if (process.platform === 'win32') {
+  spawnCommand('ls-lint-windows.exe');
+} else {
+  console.log("Unsupported OS");
```

Comment:
we should use a error message like: 

console.log('ls-lint builds are not available on platform: ' + process.platform)